shared/js/background/atb.js

```diff
@@ -175,6 +243,11 @@ const ATB = (() => {
                         return !!atb
                     })
 
+                    // in case there is no assigned atb, enroll into Chrome Counteract experiment
+                    if (!atb) {
```

Comment:
As Jonathon mentioned, you probably want to filter this to chrome only.
```suggestion
                    if (!atb && utils.getBrowserName() === 'chrome') {
```